mailbox/store/src/test/java/org/apache/james/mailbox/store/mail/model/ListMessagePropertiesAssert.java

```diff
@@ -28,14 +28,14 @@
 import com.google.common.base.Objects;
 
 
-public class ListMessagePropertiesAssert {
```

Comment:
this one should still be public, no?

Reply:
It seems to be used by `ListMessagePropertiesAssertTest` and `MessageMapperTest` only, that are part of the same package as `ListMessagePropertiesAssert`, so I thought it could be package private. What do you think it still should be public?

Reply:
- some methods of it are public, so it would make it more uniform
- it's a general use class test, that could be used by other classes of other packages, I don't see any reason to have it package-private.

Reply:
Applying the same principle to `ListMailboxAssert`, `ListMessageAssert`, `ListMessagePropertiesAssert`, `MetadataMapAssert` that follow this logic

Reply:
oh thanks, I missed them